libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -168,6 +169,14 @@ bool AP_RangeFinder_LightWareI2C::sf20_send_and_expect(const char* send_msg, con
     return memcmp(rx_bytes, expected_reply, expected_reply_len) == 0;
 }
 
+const char *AP_RangeFinder_LightWareI2C::get_version() const {
+    char *full_version = (char*) malloc((LIDAR_VERSION_STRING_SIZE + strlen(RANGEFINDER_LIDAR_I2C_VERSION_PREFIX))
```

Comment:
This will cause a memory leak. I suggest the arguments passing in my other comment.

Reply:
Fixed!